lld/wasm/Driver.cpp

```diff
@@ -467,6 +467,9 @@ getBuildId(opt::InputArgList &args) {
 
 // Initializes Config members by the command line options.
 static void readConfigs(opt::InputArgList &args) {
+  config->allowMultipleDefinition =
+      args.hasFlag(OPT_allow_multiple_definition,
+                   OPT_no_allow_multiple_definition, false);
```

Comment:
It looks like ELF also allows the -z form of this option:

```
  config->allowMultipleDefinition =                                              
      hasZOption(args, "muldefs") ||                                             
      args.hasFlag(OPT_allow_multiple_definition,                                
                   OPT_no_allow_multiple_definition, false);   
```

Maybe we should do that same, for parity?

Reply:
`-z muldefs` added and tested same as ELF.